src/components/common/navbar/Navbar.js

```diff
@@ -45,54 +45,88 @@ class Navbar extends Component {
     } = this.props
 
     return (
-      <Grid.Row className="noPadding">
+      <div className="navbar">
         {head}
 
-        <Grid.Column className="navbar noPadding">
-          <Menu borderless as="nav">
-            <Menu.Menu className={search ? 'sideAreaWithSearch' : 'sideAreaWithoutSearch'}>
-              <Menu.Item icon active={sidebarVisible} name="sidebar" onClick={handleSidebarToggle}>
-                <Icon name="sidebar" />
-              </Menu.Item>
-              <Menu.Header as="h1" className="navbarTitle" content={title} />
-            </Menu.Menu>
+        <div className="sideArea">
+          <Menu borderless className="noBorder">
+            <Menu.Item icon active={sidebarVisible} name="sidebar" onClick={handleSidebarToggle}>
+              <Icon name="sidebar" />
+            </Menu.Item>
+            <h1>
```

Comment:
Do you wanna have such small statements padded generally? Don't you think one line would be enough? :)